boot/boot.S

```diff
@@ -22,7 +22,7 @@ init_dl: .byte 0
 
 	PRINT_STR $protecte_msg, $23
 	mov %cr0, %eax
-	mov $0x1, %eax
+	or $0x1, %eax
```

Comment:
What value is `%cr0` holding at the moment that we lose by `mov`ing instead of `or`ing `%eax` ?

Reply:
in our 386 qemu model this is the value of cr0:
CR0=00000010 (which means the presence of a FPU coprocessor a 80387)